services/scraper/scraper_test.go

```diff
@@ -0,0 +1,50 @@
+package scraper_test
+
+import (
+	"fmt"
+	"net/url"
+
+	"go-google-scraper-challenge/initializers"
+	"go-google-scraper-challenge/models"
+	"go-google-scraper-challenge/services/scraper"
+	. "go-google-scraper-challenge/tests"
+	. "go-google-scraper-challenge/tests/helpers"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Scraper", func() {
+	Describe("#Search", func() {
+		Context("given valid keywords", func() {
+			BeforeEach(func() {
+				keyword := "keyword"
+				visitURL := fmt.Sprintf("http://www.google.com/search?q=%s", url.QueryEscape(keyword))
+				cassetteName := "scraper/success"
+
+				RecordResponse(cassetteName, visitURL)
+			})
+
+			It("creates a result with the given keyword", func() {
```

Comment:
Is it possible to have a test to check scraping result?

Reply:
@gutakk Added the test to check the result in [2f2fd3a](https://github.com/carryall/go-google-scraper-challenge/pull/40/commits/2f2fd3a9d30a3747a3fb465190de4068cd76d20d)